GSA/made-in-america
# [MIA-342] Pagination Bug Fix

## Description
The pagination was not working properly. The middle numbers between 3 and the last page were not appearing. This PR  corrects that issue as all page numbers showing when user presses the next button.

## How to test

After you pull down, click through the 'next' and 'previous' tabs to see all the pages. As of this writing, there are 8 pages. 

To prepare for code review, please follow the following checks.
